### DISCUSSION TITLE
AboutTheGarden

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#9] [备发布站的评论系统]
# AboutTheGarden

hello👋，我是oldwinter❄️，写过几万行代码，带过十几人团队。

https://notes.oldwinter.top/

**Comment by oldwinter** [comment]
看一下嵌入评论系统的效果。欢迎大家留言~~~

**Reply by Halecoder** [reply]
大佬，双向链接无法引用某一个标题吗，只能是文章？

**Reply by oldwinter** [reply]
没试过哦，可能得去quartz的discord社区提需求讨论了。。我几乎没有标题引用和块引用的需求现在。。

**Reply by Halecoder** [reply]
最后一个问题，我看您的dg里能够实现不点击链接就能上下文阅读，而dg3不行，是因为我的原因还是没有添加这个功能

**Comment by bianyeyu** [comment]
哇，刚刷到发布方案，从那儿跳过来，支持一下

**Comment by wintersdog** [comment]
学习了，这个双链笔记分享库真的很棒，感谢作者👍~~

**Comment by luolin1024** [comment]
请问一下作者，这个库具体是怎么做到发布的
<img width="649" alt="image" src="https://user-images.githubusercontent.com/35687278/185743517-d57a2abb-8383-4cf1-9dcd-8a2da0cf2645.png">

**Reply by oldwinter** [reply]
前几天刚挖的坑，正在写呢~~你来的正好，想请问一下，garden.oldwinter.top 和notes.oldwinter.top这2个网站，你更喜欢哪个？我根据你们的想法优先挑一个把教程补齐全一点。

**Reply by myoursh** [reply]
garden.oldwinter.top 这个发布页效果好棒！请问有没有汉化教程呢？

**Reply by oldwinter** [reply]
这个基于开源的quartz搞的。他的作者还在积极更新和维护。我看到前几天做了汉化国际化，你有能力可以直接fork他的最新代码，已经有中文汉化咯。然后遇到啥部署问题可以参考我的dg3仓库修改。

**Reply by Ravenclawer** [reply]
我直接fork了您的dg3，但用netlify部署失败了，可能是缺了文件吧，quarz有搜索功能，感觉还蛮需要的

**Reply by Ravenclawer** [reply]
我之前都是用gridea，但那个没办法双向链接

**Reply by Ravenclawer** [reply]
其实想想如果要注重explore这个概念的话，搜索的优先级可能没有那么高，没有搜索可以强制访客去慢慢探索

**Reply by Halecoder** [reply]
。。。为啥楼上消息会发到我邮箱里来

**Reply by Ravenclawer** [reply]
这个相当于github的issue，评论过就会收到消息

**Reply by Halecoder** [reply]
哦哦。。。大佬，我用的是vercel,只能托管静态页面的

**Reply by Halecoder** [reply]
用github action将md文件转为html到gitpage，然后vercel托管gitpage

**Reply by Halecoder** [reply]
作者也是这样干的

**Reply by Ravenclawer** [reply]
您才是大佬，我都不知道您在说什么

**Reply by Ravenclawer** [reply]
我之前没有用过git action 把md转为html

**Reply by Ravenclawer** [reply]
我之前是用action直接发布成github页面，但访问非常的慢

**Reply by Halecoder** [reply]
就是您说的这种，转成github页面，然后vercel托管这个页面即可

**Reply by Halecoder** [reply]
等我找个教程

**Reply by Halecoder** [reply]
https://ob.tianzhongs.ml/001_%E9%83%A8%E7%BD%B2Obsidian%E9%9D%99%E6%80%81%E7%9F%A5%E8%AF%86%E5%BA%93%E7%BD%91%E7%AB%99 看看这位写的

**Comment by wangyunzi** [comment]
大佬牛逼啊。真好看这个
